src/state/elementsVisibilityAC.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable object-shorthand */
+export const SIDEBAR_VISIBILITY = 'SIDEBAR_VISIBILITY';
+export const RESULT_SCREEN_VISIBILITY = 'RESULT_SCREEN_VISIBILITY';
+export const SW_MODALE_VISIBILITY = 'SW_MODALE_VISIBILITY';
+export const IS_CARD_FLIPPED = 'IS_CARD_FLIPPED';
+export const NETWORK_INDICATOR_VISIBILITY = 'NETWORK_INDICATOR_VISIBILITY';
+
+interface SidebarVisibilitySetAC {
```

Comment:
Might we should apply next changes:

`type SidebarVisibilitySetAC { type: 'SidebarVisibilitySetACAction', payload: boolean }`

and other similar